File: TECBank Movil/app/src/main/java/com/example/app/API/Cuenta.kt
package com.example.app.API
// Clase encargada de administar los datos serializados de la cuenta en el API
data class Cuenta(
    val numero_cuenta: String,
    val descripcion: String,
    val moneda: String,
    val tipo_cuenta: String,
    val acliente: String
)

File: RestAPI/Catalog/Entities/cuenta.cs
using System;
namespace Catalog.Entities
{
    public record Cuenta  //Clase interna de las Cuentas  dentro del api
    { 
        public Guid numero_cuenta {get; init;} //Init: Solo le permitimos asignar un valor durante su inicializacion

        public string descripcion {get; init;}

        public string moneda {get; init;}

        public string tipo_cuenta {get; init;}

        public string Acliente {get; init;}

    }
}